radicle-crypto/src/ssh/keystore.rs

```diff
@@ -82,7 +86,10 @@ impl Keystore {
 
     /// Load the secret key from the store, decrypting it with the given passphrase.
     /// Returns `None` if it wasn't found.
-    pub fn secret_key(&self, passphrase: &str) -> Result<Option<SecretKey>, Error> {
+    pub fn secret_key(
+        &self,
+        passphrase: Passphrase,
+    ) -> Result<Option<Zeroizing<SecretKey>>, Error> {
```

Comment:
Any reason why you didn't opt to implement/derive Zeroizing for `SecretKey`?

Reply:
Yeah, I tried that first actually but couldn't get it working; you can give it a spin if you want.